.github/workflows/kernel.yml

```diff
@@ -95,45 +67,51 @@ jobs:
           echo "SUBTARGET=$SUBTARGET" >> "$GITHUB_ENV"
 
       - name: Update & Install feeds
+        shell: su buildbot -c "sh -e {0}"
+        working-directory: openwrt
```

Comment:
smooth!

Reply:
Yep no idea if we should use gosudo or this to change the entire shell...imho this is more safe